src/TriggerBinding/SqlTableChangeMonitor.cs

```diff
@@ -491,8 +491,9 @@ private async Task ReleaseLeasesAsync(SqlConnection connection, CancellationToke
             await this._rowsLock.WaitAsync(token);
             this._logger.LogDebugWithThreadId("END WaitRowsLock - ReleaseLeases");
             long newLastSyncVersion = this.RecomputeLastSyncVersion();
+            bool retrySucceeded = false;
 
-            try
+            for (int retryCount = 1; retryCount <= MaxRetryReleaseLeases && retrySucceeded == false; retryCount++)
```

Comment:
Replace `retrySucceeded == false` with `!retrySucceeded`. In general, the pattern where Boolean value is compared against `true` or `false` is discouraged in all programming languages.

Reply:
Thanks, changed it.